ga4gh/datamodel/variants.py

```diff
@@ -6,11 +6,15 @@
 from __future__ import print_function
 from __future__ import unicode_literals
 
+import atexit
 import datetime
 import glob
+import json
 import os
+import os.path
```

Comment:
Speaking of which, check out this thread

http://stackoverflow.com/questions/101268/hidden-features-of-python